base/pom.xml

```diff
@@ -160,8 +160,11 @@ of Jackson: application code should only rely on `jackson-bom`
               under "META-INF/versions/11" (instead of root, /); helps pre-Java9
               libraries, frameworks, as well as avoids warnings by tooling
             -->
+	  <!-- 27-Jan-2022, tatu: as per [databind#3380] etc, really need to use
```

Comment:
Tab/space mixup, if I'm to believe the Github UI:
```suggestion
          <!-- 27-Jan-2022, tatu: as per [databind#3380] etc, really need to use
```

Reply:
Ah thanks! Dang Emacs.